_episodes/038-jon-gjengset.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Rust for Rustaceans by Jon Gjengset"
+date: 2021-09-24T23:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e038-jon-gjengset.mp3
+duration: "1:21:48"
+length: "79237006"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with Jon Gjengset, a software enginner at [AWS](https://aws.amazon.com/), about his book [Rust for Rustaceans](https://nostarch.com/rust-rustaceans).
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+
+- [@01:25] - Jon's Bio 
+- [@11:24] - [Noria](https://github.com/mit-pdos/noria)
+- [@13:00] - [Jon's Youtube Channel](https://www.youtube.com/c/JonGjengset)
+- [@21:53] - [Crust of Rust](https://www.youtube.com/watch?v=rAl-9HwD858&list=PLqbS7AVVErFiWDOAVrPt7aYmnuuOLYvOa)
+- [@25:13] - What does it mean to be a Rustacean?
+   - [Niko Matsakis' Rustacean Principles](https://github.com/nikomatsakis/rustacean-principles)
+- [@27:23] - What does intermediate content mean?
+- [@30:03] - [Rust memory model](https://nostarch.com/download/samples/RustforRustaceans_Ch2new.pdf)
```

Comment:
```suggestion
- [@30:03] - [Chapter on Memory in Rust](https://nostarch.com/download/samples/RustforRustaceans_Ch2new.pdf)
```
"Memory model" has a very particular meaning related to concurrency that doesn't apply here.